cluster/aws.go

```diff
@@ -30,6 +29,7 @@ import (
 	"io/ioutil"
 	"os"
 	"strings"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Doc is missing from all public functions

Reply:
reset to private